### ISSUE TITLE
Internationalize / localize the app?

### ISSUE THREAD

**Issue by lazerwalker** [flappy-royale/flappy-royale#92]
The pixel font we're using has accented ASCII characters, which means doing EFIGS is at least technically possible.

Definitely not for v1, but worth thinking about in case this gets any traction.

**Comment by orta** [comment]
Yeah, I'd like this, we don't have much copy, so it should be reasonably easy

**Comment by lazerwalker** [comment]
Do you have a JS localization library you've used before?

Rolling our own would be fairly straight-forward for our use case, but that also seems a bit silly.

**Comment by orta** [comment]
Never, new to me!

**Comment by lazerwalker** [comment]
If we want to use a library, https://github.com/azerion/phaser-i18next seems reasonable

**Comment by lazerwalker** [comment]
To think about: the 3 main menu buttons (Royale / Daily Trial / Settings) are currently the only non-font text we have. This'll be easier if we can figure out a way to get the same result for those without having to generate backgrounds for each language.
